### DISCUSSION TITLE
GitHub download failing intermittently from multiple office locations | potential restriction on Github for simultaneous connections from same public NAT IP

### DISCUSSION THREAD

**Discussion by Stephengopal** [community/community#138223] [Other Feature Feedback, Questions, & Ideas]
We recently started facing intermittent download failure with below errors from multiple office locations.
It looks to me that there may be some sort of restriction on Github for the requests initiated from multiple systems with the same Public IP being used for the enterprise network.

git clone https://github.com/SchedMD/slurm.git

Cloning into 'slurm'...
remote: Enumerating objects: 706119, done.
remote: Counting objects: 100% (19570/19570), done.
remote: Compressing objects: 100% (5153/5153), done.
error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
error: 6610 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

Could some one please share your thoughts and suggestions for the above issue?

**Comment by MasterInQuestion** [comment]
See also: https://github.com/orgs/community/discussions/138222

    I think alike matter most for "push" alike?
    For mere downloading, haven't ever experienced alike.
    (GitHub tends to be always online)

    Could you access the web interface normally meanwhile?
    E.g. https://github.com/SchedMD/slurm/archive/refs/heads/master.zip

    The error message indicated somewhat user-end network interference.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Stephengopal ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Stephengopal** [reply]
Thank you so much!
